src/lib.rs

```diff
@@ -233,10 +228,25 @@ fn clockpoly(
     Polygon::new(LineString::from(arcs), vec![])
 }
 
+/// Returns the first `n` [triangular numbers](https://en.wikipedia.org/wiki/Triangular_number),
+/// excluding the 0th.
+pub fn triangular_sequence(n: usize) -> Vec<f64> {
+    (1..=n).map(|i| 0.5 * (i as f64) * (i + 1) as f64).collect()
```

Comment:
That is a very concise and clever triangular number generator, think we could use that in the R version!

Reply:
In fact we use the inbuilt R function to calculate the cumulative sum:

```r
zb_100_triangular_numbers = cumsum(1:100)
```

then we use it like this https://github.com/zonebuilders/zonebuilder/blob/master/R/helper_functions.R#L44